tools/barista/src/builder/overview.ts

```diff
@@ -41,6 +43,13 @@ const highlightedItems = [
   'Theming',
 ];
 
+const navigationOrder = new Map([
```

Comment:
Not sure about this solution for ordering, because as every menu item is present in this ordering map, why do we even read the directories?